client/src/NewEvent.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+
+const NewEvent: React.FC = () => {
+    const [title, setTitle] = useState("");
+    const [location, setLocation] = useState("");
+    const [dates, setDates] = useState<string[]>([""]);
+
+    const handleSubmit = (e: React.FormEvent) => {
+        e.preventDefault();
+        const payload = {
+            name: "Team building",
+            location,
+            title,
+            dates: dates.map((d) => new Date(d).getTime()),
+        };
+
+        fetch("/api/events", {
```

Comment:
pozor, pokud nevyplnil data a jen to odeslu, odesle se to jako null :)

<img width="2520" height="1538" alt="image" src="https://github.com/user-attachments/assets/8dfe5c76-6603-4540-ae95-f5dc857596c0" />